### DISCUSSION TITLE
Multiple Dynamic Bot

### DISCUSSION THREAD

**Discussion by saadhossain** [bukhalo/nestjs-telegraf#1223] [Q&A]
I have multiple bot tokens in my database... I would like to fetch them and run multiple bots, one bot for each token..
Is there any solution for this?

**Comment by damir-minkashev** [comment]
@saadhossain Try this [sample/01-complete-app/src/app.module.ts](https://github.com/evilsprut/nestjs-telegraf/blob/main/sample/01-complete-app/src/app.module.ts)

**Comment by bukhalo** [comment]
See https://github.com/robot-mafia/nestjs-telegraf/discussions/1212#discussioncomment-10122699
